parser_library/test/workspace/instruction_sets_test.cpp

```diff
@@ -101,9 +101,10 @@ std::string sam31_macro = R"( MACRO
         MEND)";
 
 const char* sam31_macro_path = is_windows() ? "lib\\SAM31" : "lib/SAM31";
-std::string hlasmplugin_folder = is_windows() ? ".hlasmplugin\\" : ".hlasmplugin/";
+std::string hlasmplugin_folder = "/.hlasmplugin";
```

Comment:
This brings up an important issue - could you draft a doc update that describes how we are going to handle different cases - paths, relative and absolute, URIs, again relative and absolute?
Will we be able to handle paths like these eclipse/che-che4z-lsp-for-hlasm#221, or will we have to convert them when e.g. reading configuration files?